multidim-interop/impl/js/v0.46/package.json

```diff
@@ -1,19 +1,19 @@
 {
   "dependencies": {
-    "@libp2p/multidim-interop": "1.0.0-c4eff4c5"
+    "@libp2p/multidim-interop": "1.0.1"
   },
   "overrides": {
     "@chainsafe/libp2p-noise": "13.0.1",
     "@chainsafe/libp2p-yamux": "5.0.0",
     "@libp2p/mplex": "9.0.5",
-    "@libp2p/tcp": "8.0.5",
```

Comment:
Are all these overrides still necessary? Even with a lock file?

Reply:
They aren't necessary for functionality but rather they are there to explicitly state the versions of the dependencies being used, without requiring one to search the lockfile.

Reply:
That sounds like they should be `"dependencies"` and not `"overrides"`?

Reply:
They are dependencies of the dependency being installed i.e. `@libp2p/multidim-interop`  and according to [npm docs](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides) since we are making sure that the same version of a package is used everywhere,  `overrides` is recommended.

Reply:
Generally letting your dependencies manage the versions of the modules they depend on is recommended, otherwise it becomes very hard to reason about what is going on when things go wrong.